client/src/index.jsx

```diff
@@ -2,25 +2,73 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import axios from 'axios';
+import BundleSelect from './components/BundleSelect.jsx';
+import BundleImages from './components/BundleImages.jsx';
 
 
 class App extends React.Component {
   constructor() {
     super();
-    this.state = {};
+    this.state = {
+      products: null,
+      currentProduct: null,
+      imageVisibility: null,
```

Comment:
as i remeber the page has it to where there is a checkbox and you can click to add to the "bundle" maybe you could make the imagevisibility property have some subproperties to  be able to manipulate which images you can see in the bundle?